src/stratus-application/src/main/java/stratus/config/CommunityRestMvcConfigurer.java

```diff
@@ -151,6 +147,7 @@ public void configurePathMatch(PathMatchConfigurer configurer) {
         //Force MVC to use /restng endpoint. If we need something more advanced, we should make a custom PathHelper
         configurer.setUrlPathHelper(new GeoServerUrlPathHelper());
         configurer.getUrlPathHelper().setAlwaysUseFullPath(true);
+        //configurer.setUseRegisteredSuffixPatternMatch(true);
```

Comment:
An additional comment indicating why these are commented out would be nice

Reply:
The whole line is new.  I was testing several different configurations to work around the new file extension bug.  Removing it now.